specification/mpcnetworkfunction/resource-manager/Microsoft.MobilePacketCore/preview/2023-05-15-preview/networkfunction.json

```diff
@@ -0,0 +1,778 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft.MobilePacketCore Network Function resource",
+    "version": "2023-05-15-preview",
+    "description": "Mobile Packet Core Network Function Management API",
+    "x-typespec-generated": [
+      {
+        "emitter": "@azure-tools/typespec-autorest"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "host": "management.azure.com",
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "flow": "implicit",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "tags": [
+    {
+      "name": "Operations"
+    },
+    {
+      "name": "NetworkFunctions"
+    }
+  ],
+  "paths": {
+    "/providers/Microsoft.MobilePacketCore/operations": {
```

Comment:
>Microsoft.MobilePacketCore [](http://example.com/codeflow?start=16&length=26)

Did you get the review and approval done for your new RP Namespace name? See: [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#resource-provider-and-type-names-best-practices](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#resource-provider-and-type-names-best-practices).

Reply:
@visingla-ms 
Yes, we have got the **ARMSignedOff** for the Namespace and the Swagger APIs. Please refer the below PR
https://github.com/Azure/azure-rest-api-specs-pr/pull/12558